src/rating-calculator/components/CandidatesChartRecords.tsx

```diff
@@ -32,6 +33,7 @@ const COLUMNS: ReadonlyArray<ColumnType> = [
   ColumnType.ACHIEVEMENT,
   ColumnType.NEXT_RANK,
   ColumnType.NEXT_RATING,
+  ColumnType.NEW_RATING,
```

Comment:
I have a suggestion.

Instead of adding a new column, let's update the existing column to contain absolute rating and the delta.

Old value: `+12`
New value: `288 (+12)`

if we apply this change, we can remove

- ColumnType.NEW_RATING
- compareSongsByNewRating function

but we'd need to increase the width for the NEXT_RATING column

Reply:
I like this approach a lot more - it is intuitive and works similarly to in-game when an upscore occurs. Let me work on that and refactor most this PR

Reply:
Will test around increasing the px in the 10-20 range, but it seems like it should be fine

Reply:
Increased by 25px - your approach looks good. I will update the PR description as a new column is no longer required